plugins/web/src/server_command.cpp

```diff
@@ -113,16 +113,22 @@ request_dispatcher_actor::behavior_type request_dispatcher(
     [self](atom::internal, atom::request, restinio_response_ptr& response,
            const rest_endpoint& endpoint, system::rest_handler_actor handler) {
       // TODO: Also consider params in the request body here.
-      restinio::query_string_params_t string_params
-        = restinio::parse_query(response->request()->header().query());
+      auto query_params = std::optional<restinio::query_string_params_t>{};
+      try {
+        query_params
+          = restinio::parse_query(response->request()->header().query());
+      } catch (restinio::exception_t& e) {
+        return response->abort(
+          400, fmt::format("failed to parse query parameters: {}\n", e.what()));
```

Comment:
Why do we need a newline at the end?

Reply:
It makes for a more pleasant user experience when using `curl` (or any command-line tool) instead of a browser.